### PR TITLE
[Fixes #6534] Missing translation tags in layer_list and layer_detail

### LINKED ISSUE #6534: Missing translation tags in layer_list and layer_detail
## Expected Behavior
In all languages, the name and description of the data categories should be displayed in the appropriate language.

## Actual Behavior

(missing {% trans item.category__gn_description} in _resourcebase_snippet.html e.g.)

## Steps to Reproduce the Problem

  1. Change language, e.g. to french
  2. Go to layer list or go to layers detail page

Bildschirmfoto von 2020-10-19 11-49-09
Bildschirmfoto von 2020-10-19 11-50-04

## Specifications

  - GeoNode version: 3.x
  - Installation method: SPCGeoNode Docker
  - Platform: Ubuntu
  - Additional details:

### PR DESCRIPTION
**Pull request by gannebamm**
Iso-Category texts will not get translated throughout the application. This PR fixes the fixtures (done) and will add `{% trans %}` tags were needed (not done yet)

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in the documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemption is, changes not visible to end-users)
- [x] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [ ] New unit tests have been added covering the changes, unless there is an explanation on why the tests are not necessary/implemented
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [ ] This PR passes the QA checks: flake8 geonode
- [ ] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

**Comment by gannebamm** [comment]
I merged current master in my branch and updated the (german) translations again. I am still stuck with getting translated iso-categories from tastypie. Therefore the search side-bar categories are not yet translated (see https://github.com/GeoNode/geonode/issues/6534 second picture, left-hand side). The other missing strings do now work (tested for the german translation).

@afabiani I would wait for the tests and then merge this. I changed the fixtures as discussed. The other issue is most likely soon obsolete due to the new v2 api? @t-book told me I could write a serializer for tastypi to get the strings translated, but that is to much work, for a (for me at least) minor issue. What do you think?

**Comment by afabiani** [comment]
@gannebamm before merging this one two things:

1. The PR includes the removal of grappelli, is this wanted?
2. The messages should be regenerated for all languages imho, even if not providing translations for all of them of course

```
DJANGO_SETTINGS_MODULE=geonode.local_settings django-admin makemessages --all -d djangojs -e "js" -i docs -i node_modules -i lib

DJANGO_SETTINGS_MODULE=geonode.local_settings django-admin makemessages --all -d django -e "html,txt,py" -i docs

```

**Comment by gannebamm** [comment]
@afabiani 
* The PR includes the removal of grappelli, is this wanted?

No it is not. I am sorry. Some local commit must have been accidently pushed to my branch on github.

* The messages should be regenerated for all languages imho, even if not providing translations for all of them of course

There are no new msgids so I think this is not needed. Nonetheless I have used the commands but do not see any local changes after `git status`. But I see some warnings for the monitoring js files, like:
```
./geonode/monitoring/frontend/monitoring/src/containers/app/actions.js:88: warning: RegExp literal terminated too early
./geonode/monitoring/frontend/monitoring/src/pages/alert-config/actions/set.js:80: warning: RegExp literal terminated too early
./geonode/monitoring/frontend/monitoring/src/pages/alerts-settings/actions.js:81: warning: RegExp literal terminated too early
./geonode/monitoring/frontend/src/api/index.js:155: warning: unterminated string
./geonode/monitoring/frontend/src/utils/RouteUtils.js:28: warning: RegExp literal terminated too early
```
But I think those can be ignored

**Review by afabiani — APPROVED** [review]

**Pull request merged**
